lua/core/norns.lua

```diff
@@ -170,6 +170,14 @@ else
 end
 
 
+-- fetch (git clone)
+norns.fetch = function(url)
+  local status = os.execute("cd "..os.getenv("HOME").."/dust/code; git clone "..url)
```

Comment:
would recommend using the `paths` module here (i think the root dir for code is available) instead of constructing the path manually

Reply:
aha thank you for reminding me!